File: VB/VB/WebSample.Module/Person.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.Xpo

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation
Imports DevExpress.ExpressApp.SystemModule
Imports DevExpress.Data.Filtering

Namespace WinSample.Module
	<DefaultClassOptions> _
	Public Class Person
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private _Name As String
		Public Property Name() As String
			Get
				Return _Name
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Name", _Name, value)
			End Set
		End Property
		Private _Address As String
		Public Property Address() As String
			Get
				Return _Address
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Address", _Address, value)
			End Set
		End Property
		Private _Country As String
		Public Property Country() As String
			Get
				Return _Country
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Country", _Country, value)
			End Set
		End Property
	End Class
End Namespace

File: CS/WebSample/WebSample.Module/Person.cs
using System;

using DevExpress.Xpo;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;
using DevExpress.ExpressApp.SystemModule;
using DevExpress.Data.Filtering;

namespace WinSample.Module {
    [DefaultClassOptions]
    public class Person : BaseObject {
        public Person(Session session) : base(session) { }
        private string _Name;
        public string Name {
            get { return _Name; }
            set { SetPropertyValue("Name", ref _Name, value); }
        }
        private string _Address;
        public string Address {
            get { return _Address; }
            set { SetPropertyValue("Address", ref _Address, value); }
        }
        private string _Country;
        public string Country {
            get { return _Country; }
            set { SetPropertyValue("Country", ref _Country, value); }
        }
    }
}
